packages/ui/src/views/repo/components/commits-list.tsx

```diff
@@ -39,9 +41,10 @@ export const CommitsList: FC<CommitProps> = ({ data }) => {
 
                   return (
                     <StackedList.Item
-                      className="!cursor-default items-start py-3"
+                      className="items-start py-3"
                       key={commit?.sha || repo_idx}
                       isLast={commitData.length - 1 === repo_idx}
+                      onClick={() => navigate(`${commit?.sha?.substring(0, 7)}`)}
```

Comment:
Lets use the new route definitions here?

Reply:
```
navigate(${routes.toRepoFiles({ spaceId, repoId })}/${commit?.sha?.substring(0, 7)})
```

Reply:
even better, can we change this to a link?